Add Elapsed Time to Timer

claire50 opened the pull request:
Fixes Issue #12

Commit:
Add Elapsed Time to Timer

owenca commented on TimerView.cpp:
```diff
@@ -5,7 +5,8 @@
 	BTextView("timer"),
-	fRunning(false)
+	fRunning(false),
+	elapsedTime(0)
```
```suggestion
	fElapsedTime(0)
```

owenca replied:
From [Haiku Coding Guidelines](https://www.haiku-os.org/development/coding-guidelines/):
![identifiers](https://user-images.githubusercontent.com/6434360/49419505-fcba5080-f73b-11e8-8b12-d27c42512a7e.PNG)

owenca commented on TimerView.cpp:
```diff
@@ -30,16 +31,17 @@ bool TimerView::Running()
 void TimerView::Stop()
 {
 	fRunning = false;
+	elapsedTime = Elapsed();
```
```suggestion
	fElapsedTime = Elapsed();
```

owenca commented on TimerView.cpp:
```diff
@@ -30,16 +31,17 @@ bool TimerView::Running()
 void TimerView::Pulse()
 {
-	bigtime_t elapsed = Elapsed();
+	bigtime_t elapsed = fRunning ? Elapsed() : elapsedTime;
```
```suggestion
	bigtime_t elapsed = fRunning ? Elapsed() : fElapsedTime;
```

owenca commented on TimerView.h:
```diff
@@ -19,6 +19,7 @@ class TimerView : public BTextView {
 private:
 	bool fRunning;
 	bigtime_t fStartTime;
+	int elapsedTime;
```
```suggestion
	int fElapsedTime;
```

owenca reviewed (changes requested):
Works for me! Just need minor changes for Haiku's naming conventions.

Commit:
Stop the timer when the game finishes

Commit:
Stop the timer when the game finishes (fixes)

claire50 commented:
Hi! I have changed the code a little bit, since I found a simpler way to fix the issue. I was having problems with committing onto GitHub, but I recloned it and changed the code again, so now it works. Thanks!

owenca commented:
Thanks for making the fix even better!
